master/internal/trial_workload_sequencer.go

```diff
@@ -281,6 +254,14 @@ func (s trialWorkloadSequencer) Workload() (workload.Workload, error) {
 	}
 }
 
+func (s trialWorkloadSequencer) hasSearcherValidation() bool {
```

Comment:
Is the plan to implement the searcher API and then move this logic to there?

Reply:
This logic will be in the python workload sequencer once it is implemented.